src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -0,0 +1,55 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.ProductDto;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("/products")
```

Comment:
Also not sure, but in materials we always used additional folder for example "v1/products" but again it's just nut picking

Reply:
We all should stick to one path, for me just /desired_path is ok.

Reply:
I think we all should add an API version for consistency